src/components/manage/UniversalLink/__snapshots__/UniversalLink.test.jsx.snap

```diff
@@ -30,7 +30,7 @@ exports[`UniversalLink renders a UniversalLink component if no external(href) li
 
 exports[`UniversalLink renders a UniversalLink component with external link 1`] = `
 <a
-  className={null}
+  className=" external"
```

Comment:
see?

Reply:
updated the snapshot. Please review it once again.